src/kinvey.d.ts

```diff
@@ -319,7 +319,7 @@ export namespace Kinvey {
     static logout(options?: RequestOptions): Promise<void>;
     signup(data: {}, options?: RequestOptions): Promise<this>;
     static signup(data: {}, options?: RequestOptions): Promise<User>;
-    update(data: {}, options?: RequestOptions): Promise<this>;
+    update(data?: {}, options?: RequestOptions): Promise<this>;
```

Comment:
Can you explain why you put this on `update()`? I am thinking you meant to put it on `signup()`.

Reply:
Yes my bad,
i meant to put it on signup